compiler/scenario-service/client/src/DA/Daml/LF/ScenarioServiceClient/LowLevel.hs

```diff
@@ -121,9 +121,13 @@ instance NFData Error where
     rnf = rwhnf
 
 findServerJar :: IO FilePath
-findServerJar = do
-  runfilesDir <- locateRunfiles (mainWorkspace </> "compiler/scenario-service/server")
-  pure (runfilesDir </> "scenario-service.jar")
+findServerJar = locateResource Resource
+  { runfilesPath = mainWorkspace </> "compiler" </> "scenario-service" </> "server" </> "scenario-service.jar"
```

Comment:
Just curious:
What does splitting the path, and recombining with `</>` achieve?

Reply:
The only difference is really that `</>` translates to a backslash `\` when running on Windows, but tbh that this hasn't been a problem so far means that the functions downstream know how to handle forward slashes `/` on Windows. Also, personally, I find the version with `</>` easier to read.